torch/csrc/jit/script/compiler.cpp

```diff
@@ -908,29 +908,61 @@ struct to_ir {
 
   Value* emitTernaryIf(const TernaryIf& expr) {
     Value* cond_value = emitCond(expr.cond());
+    auto first_expr = [&] {
+      return emitExpr(expr.true_expr());
+    };
+    auto second_expr = [&] {
```

Comment:
nit: `true_expr` and `false_expr` are much better names than `first` and `second`

Reply:
agreed